libexec/scoop-config.ps1

```diff
@@ -123,25 +127,27 @@
 
 param($name, $value)
 
-. "$psscriptroot\..\lib\core.ps1"
-. "$psscriptroot\..\lib\help.ps1"
+. "$PSScriptRoot\..\lib\core.ps1"
+. "$PSScriptRoot\..\lib\help.ps1"
 
 reset_aliases
 
-if(!$name) { my_usage; exit 1 }
-
-if($name -like 'rm') {
+if (!$name) {
+    $scoopConfig
+} elseif ($name -like '--help') {
+    my_usage
+} elseif ($name -like 'rm') {
```

Comment:
```suggestion
} elseif ($name -like 'rm' -or $null -eq $value) {
```

What do you think of letting `scoop config <config> $null` to remove the config too?

Reply:
Hmm, I think it would get confusing. Both `scoop config rm hello` and `scoop config hello $null` would be redundant. Also $null is not a recognized symbol in CMD/Bash?

Reply:
Okay, yes, in BASH/CMD `$null` may not be recognized.